js-exercises/abbreviate-string/abbreviateString.test.js

```diff
@@ -0,0 +1,16 @@
+import { abbreviateString } from './abbreviateString';
+
+describe('abbreviateString', () => {
+  it('abbreviates the following strings', () => {
+    expect(abbreviateString('Hacktober Fest')).toEqual('Hacktober F.');
```

Comment:
Please add test for single word as well.

```suggestion
    expect(abbreviateString('Hacktober Fest')).toEqual('Hacktober F.');
    expect(abbreviateString('Single')).toEqual('Single');
```